### PR TITLE
Update docs for android usage

### PR DESCRIPTION
**Pull request by evelant**
Add section to the docs about running on Android studio emulator

### CODE REVIEW

File: src/main/app/inspect_native_preload.cljs [hunk 1]
```diff
@@ -1,11 +1,11 @@
 (ns app.inspect-native-preload
   (:require
-    [com.fulcrologic.fulcro.inspect.inspect-client :as inspect]
```
Comment (awkay) [inline src/main/app/inspect_native_preload.cljs]: Please don't push ws changes from your indent settings

**Comment by evelant** [comment]
@awkay OK I removed the whitespace change. Can I ask why it's formatted with the extra indentation for the :require key? I'm new to clojure and I don't quite understand all the conventions yet.

I'm using [Calva](https://github.com/BetterThanTomorrow/calva/tree/master/src/calva-fmt) as my editor and it automatically follows [this style guide](https://github.com/bbatsov/clojure-style-guide#line-break-ns-declaration) which says to align the requires as in my original commit rather than indent them further. I can't find any style guide resources pointing to keeping more indent there and I can't figure out how to configure my editor to keep it -- I had to disable formatting to commit the file with the current whitespace.

**Comment by awkay** [comment]
I use IntelliJ, and that is the spacing it has set. I care less about why and more that the commit is concise :)

This is just a general problem in any programming language: people can never agree on a format. It's all personal preference, and nothing is "wrong" with what you're using.

**Comment by awkay** [comment]
Thanks for the doc update!

**Pull request merged**